spec/requests/internal/broadcasts_spec.rb

```diff
@@ -112,4 +112,19 @@
       end
     end
   end
+
+  context "with the same title and the same type_of" do
+    let(:super_admin) { create(:user, :super_admin) }
+    let(:params) { { title: "Hello!", processed_html: "<p>Hello!</p>", type_of: "Announcement", active: true } }
```

Comment:
One small thing: I think this is actually testing that we can't create two active `Announcement` broadcasts (because we have a validation on that specific case, too!) In fact, you could try this out: if you tried this test out `master`, I suspect it would pass, because of the validation that ensures that only one `Announcement` broadcast is `active` at a time :)

But!!! There are two ways around this, which will make the test more explicit:
1. remove the active: 
```suggestion
    let(:params) { { title: "Hello!", processed_html: "<p>Hello!</p>", type_of: "Announcement" } }
```
2. or, test this against `Welcome` type broadcasts.
```suggestion
    let(:params) { { title: "Hello!", processed_html: "<p>Hello!</p>", type_of: "Welcome", active: true} }
```

I only thought of this because I wrote the logic for the Announcement validation! 😸

Reply:
Whoops! Good catch. 

I've made the change!